pyangbind/helpers/misc.py

```diff
@@ -1,5 +1,5 @@
 """
-Copyright 2015, Rob Shakir 
+Copyright 2015, Rob Shakir
```

Comment:
Per the other PR, I'll go through and fix these because this is now (c) all contributors.

Reply:
No action required from author.